intel-retail/automated-self-checkout
# feat:  add capi yolov5 implementation and docs

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #323 

## Anything the reviewer should know when reviewing this PR?

It may take up to 30 minutes to make build if you have not build ovms-gst-capi before.

## Test Instructions if applicable
make build-capi_yolov5
make clean-all
make run-camera-simulator
run the pipeline: ```PIPELINE_PROFILE="capi_yolov5" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_1 --workload ovms```

upon successful you should see FPS value, may or may not have object been detected.

make run-smoke-tests

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
